easybuild/easyconfigs/j/JupyterLab/JupyterLab-3.5.0-GCCcore-11.3.0.eb

```diff
@@ -53,4 +53,9 @@ modextravars = {
     'JUPYTERLAB_DIR': '%(installdir)s/share/jupyter/lab',
 }
 
+modluafooter = """
+setenv("JUPYTERLAB_SETTINGS_DIR", pathJoin(os.getenv("HOME"), ".jupyter", "lab", "user-settings"))
+setenv("JUPYTERLAB_WORKSPACES_DIR", pathJoin(os.getenv("HOME"), ".jupyter", "lab", "workspaces"))
+"""
+
```

Comment:
@lexming is this going to offer any surprises to existing users? If so, is it worth adding a module load message about the change?

Reply:
There should be no issue.  This change just allows users to keep their own configuration for the lab. So that if they change some setting, it will be kept the next time they load the module. Currently this is not the case, and all settings are reset.

Reply:
I added a comment to the easyconfig in https://github.com/easybuilders/easybuild-easyconfigs/pull/17982/commits/92ffc4492e83bc8afb88c25f3dd06b90ff82dfc7

Reply:
I assume this would cause issues if anyone is using tcl modules?